cmd/minikube/cmd/config/open.go

```diff
@@ -110,7 +114,8 @@ You can add one by annotating a service with the label %s:%s
 func init() {
 	addonsOpenCmd.Flags().BoolVar(&addonsURLMode, "url", false, "Display the kubernetes addons URL in the CLI instead of opening it in the default browser")
 	addonsOpenCmd.Flags().BoolVar(&https, "https", false, "Open the addons URL with https instead of http")
-
+	addonsOpenCmd.Flags().IntVar(&wait, "wait", constants.DefaultWait, "Amount of time to wait for service in seconds")
+	addonsOpenCmd.Flags().IntVar(&interval, "interval", constants.DefaultInterval, "The time interval for each check that wait performs in seconds")
```

Comment:
The issue here is that, for users, I tested the timing (w/ 6 second interval) so that there were only ~3-6 outputs "waiting" if a user waited for a sensible service to come up (single pod service).  For something automated, there is no real reason to have the wait be so high.  I am fine with removing it though.